libp2p-peerstore/src/peerstore_tests.rs

```diff
@@ -29,11 +29,11 @@
 
 macro_rules! peerstore_tests {
     ({$create_peerstore:expr} $({$stmt:stmt})*) => {
+        extern crate multihash;
```

Comment:
Probably you want to put the `mod whatever` in this macro too (instead of requiring the user to write it) so you don't get obnoxious and difficult-to-handle errors if you call this twice in the same module

Reply:
Discussed IRL and agreed that we wouldn't do that because it adds an indentation level and because the JSON backend requires an additional crate.